client/components/d3chart/utils.js

```diff
@@ -458,22 +456,12 @@ export const drawAxis = ( node, params ) => {
 		.remove();
 };
 
-const getTooltipRowLabel = ( d, row, params ) =>
-	d[ row.key ].labelDate ? formatDate( params.pointLabelFormat, d[ row.key ].labelDate ) : row.key;
-
-const getFormatedTotal = ( total, valueType ) => {
-	let rowTotal = total;
-	switch ( valueType ) {
-		case 'average':
-			rowTotal = Math.round( total );
-			break;
-		case 'currency':
-			rowTotal = formatCurrency( total );
-			break;
-		case 'number':
-			break;
-	}
-	return rowTotal;
+const getTooltipRowLabel = ( d, row, params ) => {
+	const isDate =
```

Comment:
I would expect a `isDate` variable to be a bool instead of a date. Could we rename this one to something else? Maybe just `date` or `labelDate`.